### ISSUE TITLE
Please support converting to FluentValue

### ISSUE THREAD

**Issue by hongquan** [mitsuhiko/minijinja#321]
[Fluent](https://projectfluent.org/) is another localization system. It will be nice if MiniJinja support converting its Value type to [FluentValue](https://docs.rs/fluent-bundle/0.15.2/fluent_bundle/types/enum.FluentValue.html), so that we can have Jinja function to lookup translation.

**Comment by mitsuhiko** [comment]
I don't think this needs to be part of MiniJinja. You can probably build yourself a `minijinja-fluent` crate which provides tools to convert the values between the two.

**Comment by hongquan** [comment]
Ok.
